pom.xml

```diff
@@ -64,21 +64,27 @@
         <cxf.version>3.1.4</cxf.version>
         <cxf.xpp3.bundle.version>1.1.4c_6</cxf.xpp3.bundle.version>
         <jts.bundle.version>1.12_1</jts.bundle.version>
-        <karaf.version>2.4.3</karaf.version>
+        <karaf.version>4.0.4</karaf.version>
         <lux.version>1.2</lux.version>
         <opengis.bundle.version>${org.geotools.version}_1</opengis.bundle.version>
         <org.geotools.bundle.version>${org.geotools.version}_1</org.geotools.bundle.version>
         <org.geotools.version>8.4</org.geotools.version>
         <org.slf4j.version>1.7.1</org.slf4j.version>
```

Comment:
A quick search says there are 48 imports of `org.slf4j.ext`.  I'll write up another ticket to get rid of it as things are running ok using mismatched versions.